src/codegen/abi.js

```diff
@@ -59,18 +60,60 @@ module.exports = class AbiCodeGenerator {
             name = `param${index}`
           }
           paramsClass.addMethod(
-            tsCodegen.method(
+            input.get('type') === 'tuple' ?
+              tsCodegen.method(
+                `get ${name}`,
+                [],
+                tsCodegen.namedType(name).capitalize(),
+                `
+            return <${tsCodegen.namedType(name).capitalize()}>${typesCodegen.ethereumValueToAsc(
```

Comment:
* You could then reuse the `tupleClassName` here.
* The indentation of the `return` here doesn't align with the opening backtick; the closing backtick is off too.
* Can we return `return ... as ${tupleClassName}` instead of using the `<${tupleClassName}>` syntax? I feel it's less cryptic.